DATA/production/qc-async/emc.json

```diff
@@ -32,7 +32,7 @@
         "maxNumberCycles": "-1",
         "dataSource": {
           "type": "direct",
-          "query": "emcal-cells:EMC/CELLS;emcal-triggerecords:EMC/CELLSTRGR"
+          "query": "emcal-cells:EMC/CELLS:0;emcal-triggerecords:EMC/CELLSTRGR:0"
```

Comment:
Are you sure that `EMC/CELLS:0` is correct and that it should not be `EMC/CELLS/0`? I don't know, but maybe you tested it already that it works like this?

Reply:
Sorry, indeed, copy/paste error. Yes, it was tested offline but then I screwed up porting to O2DPG. Thanks for spotting! Will fix immediately!